docs/02-use-cases/06-batch-job.md

```diff
@@ -9,5 +9,5 @@ permalink: /docs/use-cases/batch-job
 A lot of batch jobs are not pure data manipulation programs. For those, the existing big data frameworks are the best fit.
 But if processing a record requires external API calls that might fail and potentially take a long time, Cadence might be preferable.
 
-One of our internal Uber customer uses Cadence for end of month statement generation. Each statement requires calls to multiple
+One of our internal Uber customers use Cadence for end of month statement generation. Each statement requires calls to multiple
```

Comment:
Hi! I appreciate the contribution to the Cadence documentation!

`customer` here is singular since we are referring to one specific team at Uber as this example.

For context, the voice of this text is the Cadence team at Uber, we refer to other teams at Uber that use Cadence as a service within Uber as internal customers.